spec/imsc-hrm.html

```diff
@@ -532,7 +535,7 @@ <h2>Paint Images</h2>
     <section id='paint-text'>
       <h2>Paint Text</h2>
 
-      <p>In the context of this section, a <dfn>glyph</dfn> is a tuple consisting of (i) one character and (ii) the computed values
+      <p>In the context of this section, a <dfn>glyph</dfn> is a tuple consisting of (i) one [[UNICODE]] code point and (ii) the computed values
```

Comment:
Consider using [=Unicode code point=] pointing to [our definition](https://w3c.github.io/i18n-glossary/#def_unicode_code_point). If you use the xref feature of respec, you can just link to our definitions (you don't need to include the [[i18n-glossary]] mention)

Reply:
Respec complains about this, quite rightly I think - it's a normative section so referencing an informative document isn't good practice. If the i18n glossary were a Recommendation, this would work really well.

Reply:
Using an xref should work okay, though?? It's a normative section, but the definition doesn't need to be normative?